ReservationSystem/LogicLayer/AdminBackEnd.cs

```diff
@@ -40,4 +40,101 @@ public static string ReadPassword()
         return password;
     }
 
+    public static void AddNewGuidedTour()
+    {
+        Console.Clear();
+        Console.WriteLine("Add New Guided Tour:");
+        Console.WriteLine("1. Add a new guided tour for today");
+        Console.WriteLine("2. Add a new guided tour to the standard schedule");
+        Console.Write("Choose an option: ");
+
+        string choice = Console.ReadLine();
+
+        switch (choice)
+        {
+            case "1":
+                AddTourForToday();
+                break;
+            case "2":
+                AddTourToStandardSchedule();
+                break;
+            default:
+                Console.WriteLine("Invalid option. Please try again.");
+                break;
+        }
+    }
+
+    private static void AddTourForToday()
+    {
+        Console.Write("Enter time for the tour (hh:mm): ");
+        string time = Console.ReadLine();
```

Comment:
make sure only the correct time format can be entered .. otherwise program crashes